lib/graphql_metrics/extractor.rb

```diff
@@ -156,20 +101,24 @@ def extract_query
     end
 
     def extract_field(irep_node)
-      return unless respond_to?(:field_extracted)
+      return unless field_extracted_method = extraction_method(:field_extracted)
       return unless irep_node.definition
 
-      field_extracted(
+      resolver_times = if instrumentor
+        instrumentor.ctx_namespace.dig(Instrumentation::TIMING_CACHE_KEY, irep_node.ast_node)
```

Comment:
I wonder if it would be cleaner to move logic like this into a method on the instrumentor, and then define a dummy instrumentor class to use instead of nil?